### ISSUE TITLE
package_data not included?

### ISSUE THREAD

**Issue by legrostdg** [scheibler/khard#231]
I'm trying to update the Debian package and noticed you added a data dir in the source. This does not get installed when I build the package... Maybe it's a Debian specific problem, but I'm not sure. Shouldn't setup.py include a package_data section?

**Comment by legrostdg** [comment]
Or maybe the data files should be added to MANIFEST.in?

**Comment by lucc** [comment]
I can not reproduce this. You will have to provide more info. I did this:
```sh
python3 -m venv test-for-231
source test-for-231/bin/activate
python3 setup.py install
tree test-for-231/lib/python3.7/site-packages/khard-0.15.0-py3.7.egg
```
which gives me
```
test-for-231/lib/python3.7/site-packages/khard-0.15.0-py3.7.egg
├── EGG-INFO
│   ├── dependency_links.txt
│   ├── entry_points.txt
│   ├── not-zip-safe
│   ├── PKG-INFO
│   ├── requires.txt
│   ├── SOURCES.txt
│   └── top_level.txt
└── khard
    ├── actions.py
    ├── address_book.py
    ├── carddav_object.py
    ├── cli.py
    ├── config.py
    ├── data
    │   ├── config.spec
    │   └── template.yaml
    ├── helpers.py
    ├── __init__.py
    ├── khard.py
    ├── __main__.py
    ├── object_type.py
    └── version.py

3 directories, 20 files
```

**Comment by legrostdg** [comment]
Thanks for trying to reproduce the issue. I'm investigating with the debian-python team to see if someone has a clue... I'll keep this updated.

**Comment by lucc** [comment]
I just released v0.16.0. The release can be found on github and pypi.
